source/speech/speech.py

```diff
@@ -947,6 +949,30 @@ def getIndentationSpeech(indentation: str, formatConfig: Dict[str, bool]) -> Spe
 	return indentSequence
 
 
+# The list containing the regions that will be shown in braille when the speak function is called
+# and the braille mode is set to speech output
+_regions: list[braille.TextRegion] = []
+
+
+def _showSpeechInBraille(speechSequence: SpeechSequence):

```

Comment:
Please make this a generic function that is braille unrelated i.e. it could provide a string of speech, but shouldn't output braille itself.code in the braille module.